### ISSUE TITLE
API :: Archive projects are pulled by default through the API; this should be turned to false

### ISSUE THREAD

**Issue by SteadyCadence** [Cadasta/cadasta-platform#1645]
We should adjust the API to **not** pull down projects that have been archived by default.

**Comment by oliverroick** [comment]
I don't think that this is a bug. You can always filter for active projects only, using `.../projects/?archived=False`.

**Comment by SteadyCadence** [comment]
I think that it makes sense to make archived projects false by default and if you want to pull archived projects, you use `.../projects/?archived=True`

**Comment by dpalomino** [comment]
Likely a QGIS bug. QGIS plugin is pulling all projects including the archived ones. To be rechecked.
